pkg/filters/builder/requestbuilder.go

```diff
@@ -57,9 +57,9 @@ type (
 
 	// RequestBuilderSpec is RequestBuilder Spec.
 	RequestBuilderSpec struct {
-		filters.BaseSpec `yaml:",inline"`
-		Spec             `yaml:",inline"`
-		Protocol         string `yaml:"protocol" jsonschema:"omitempty"`
+		filters.BaseSpec `json:",inline"`
+		Spec             `json:",inline"`
```

Comment:
Sure, but the `inline` here just indicates the embedding field, which works well for marshal and unmarshal. But remaining the tag `yaml` is good for MarshalYAML indeed. How about adding `yaml` for all inline fields?

Reply:
And there is another method in a more comfortable way: fork yaml repo and support json tag in fallback way like this PR: https://github.com/go-yaml/yaml/pull/774

Reply:
there are more cases than `inline`, like ignored fields( `-` ), and keys could also be different without the tag.
I have no idea about the fallback way, but as all the structures are defined by us, I prefer to use both tags.

Reply:
I did it in the vendor way[1] instead of a redundant way to copy tag info in massive fields across this repo. If there are indeed some fields needing to differ in json and yaml tag in the future, it will use yaml tag as the higher priority.

[1] https://github.com/megaease/yaml/blob/v3/yaml.go#L530-L537